INFO - 2024-04-13 07:19:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 07:19:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-04-13 07:19:18 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 184)
#1 C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('https: //phpmya...', 'w1836097', Object(SensitiveParameterValue), 'w1836097_0', 3306, '', 0)
#2 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#11 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#12 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#14 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('https: //phpmya...', 'w1836097', Object(SensitiveParameterValue), 'w1836097_0', 3306, '', 0)
#1 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#11 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#9 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-04-13 07:19:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for https failed: No such host is known. 
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`firstname`, `lastname`, `email`, `username`, `password`) VALUES (:firstname:, :lastname:, :email:, :username:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\User.php(52): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->register()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 07:20:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-04-13 07:20:18 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 184)
#1 C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('https: //phpmya...', 'w1836097', Object(SensitiveParameterValue), 'w1836097_0', 3306, '', 0)
#2 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#11 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#12 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#14 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('https: //phpmya...', 'w1836097', Object(SensitiveParameterValue), 'w1836097_0', 3306, '', 0)
#1 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#11 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#9 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-04-13 07:20:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for https failed: No such host is known. 
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`firstname`, `lastname`, `email`, `username`, `password`) VALUES (:firstname:, :lastname:, :email:, :username:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\User.php(52): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->register()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 07:20:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-04-13 07:20:20 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 184)
#1 C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('https: //phpmya...', 'w1836097', Object(SensitiveParameterValue), 'w1836097_0', 3306, '', 0)
#2 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#11 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#12 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#14 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('https: //phpmya...', 'w1836097', Object(SensitiveParameterValue), 'w1836097_0', 3306, '', 0)
#1 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#11 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#9 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-04-13 07:20:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for https failed: No such host is known. 
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`firstname`, `lastname`, `email`, `username`, `password`) VALUES (:firstname:, :lastname:, :email:, :username:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\User.php(52): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->register()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 07:20:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 07:20:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-04-13 07:20:33 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 184)
#1 C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('https: //phpmya...', 'w1836097', Object(SensitiveParameterValue), 'w1836097_0', 3306, '', 0)
#2 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#5 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#7 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#8 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#9 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#11 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#12 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#14 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('https: //phpmya...', 'w1836097', Object(SensitiveParameterValue), 'w1836097_0', 3306, '', 0)
#1 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#11 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for https failed: No such host is known.  in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#9 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-04-13 07:20:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for https failed: No such host is known. 
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`firstname`, `lastname`, `email`, `username`, `password`) VALUES (:firstname:, :lastname:, :email:, :username:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\User.php(52): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->register()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 07:25:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 07:25:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-04-13 07:25:46 --> Error connecting to the database: mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('phpmyadmin.ecs....', 'w1836097', Object(SensitiveParameterValue), 'w1836097_0', 3306, '', 0)
#1 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#11 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#9 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-04-13 07:25:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`firstname`, `lastname`, `email`, `username`, `password`) VALUES (:firstname:, :lastname:, :email:, :username:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\User.php(52): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->register()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 07:25:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-04-13 07:25:56 --> Error connecting to the database: mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:184
Stack trace:
#0 C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php(184): mysqli->real_connect('phpmyadmin.ecs....', 'w1836097', Object(SensitiveParameterValue), 'w1836097_0', 3306, '', 0)
#1 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#11 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\xampp\htdocs\serverCw\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\serverCw\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\serverCw\system\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\serverCw\system\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\serverCw\system\BaseModel.php(802): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\serverCw\system\Model.php(744): CodeIgniter\BaseModel->insert(Array, false)
#6 C:\xampp\htdocs\serverCw\system\BaseModel.php(713): CodeIgniter\Model->insert(Array, false)
#7 C:\xampp\htdocs\serverCw\app\Controllers\User.php(52): CodeIgniter\BaseModel->save(Array)
#8 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(941): App\Controllers\User->register()
#9 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#10 C:\xampp\htdocs\serverCw\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\serverCw\public\index.php(79): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2024-04-13 07:25:56 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2307): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`firstname`, `lastname`, `email`, `username`, `password`) VALUES (:firstname:, :lastname:, :email:, :username:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(326): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(802): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(744): CodeIgniter\BaseModel->insert([...], false)
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->insert([...], false)
 7 APPPATH\Controllers\User.php(52): CodeIgniter\BaseModel->save([...])
 8 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->register()
 9 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
10 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 07:26:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 07:26:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 07:26:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 07:27:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 08:51:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 08:52:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 08:52:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 08:52:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 10:30:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:30:52 --> Error: Call to undefined method CodeIgniter\Session\Session::questionsdata()
in APPPATH\Controllers\Questions.php on line 23.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Questions->create()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Questions))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:32:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:32:18 --> Error: Call to undefined method CodeIgniter\Session\Session::userdata()
in APPPATH\Controllers\Questions.php on line 23.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Questions->create()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Questions))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:33:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 10:38:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:38:30 --> Error: Call to undefined method CodeIgniter\Session\Session::get_current_user()
in APPPATH\Controllers\Questions.php on line 23.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Questions->create()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Questions))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:40:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 10:40:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 10:41:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 10:41:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:41:29 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 22.
 1 APPPATH\Models\UserModel.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 22)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:41:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:41:30 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 22.
 1 APPPATH\Models\UserModel.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 22)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:41:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:41:31 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 22.
 1 APPPATH\Models\UserModel.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 22)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:41:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:41:37 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 22.
 1 APPPATH\Models\UserModel.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 22)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:42:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:42:01 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 22.
 1 APPPATH\Models\UserModel.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 22)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:42:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:42:01 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 22.
 1 APPPATH\Models\UserModel.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 22)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:42:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:42:02 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 22.
 1 APPPATH\Models\UserModel.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 22)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:42:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:42:02 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 22.
 1 APPPATH\Models\UserModel.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 22)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:42:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:42:02 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 22.
 1 APPPATH\Models\UserModel.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 22)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:42:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:42:02 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 22.
 1 APPPATH\Models\UserModel.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 22)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:42:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:42:09 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 22.
 1 APPPATH\Models\UserModel.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 22)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:44:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:44:33 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 22.
 1 APPPATH\Models\UserModel.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 22)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:46:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:46:35 --> Error: Call to undefined method CodeIgniter\Session\Session::isStarted()
in APPPATH\Controllers\User.php on line 37.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:47:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:47:52 --> Error: Call to undefined method CodeIgniter\Session\Session::isStarted()
in APPPATH\Controllers\User.php on line 36.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:49:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:49:26 --> Error: Call to undefined method CodeIgniter\Session\Session::set_flashdata()
in APPPATH\Controllers\User.php on line 37.
 1 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 2 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 3 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:49:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 10:50:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 10:50:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 10:53:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:53:21 --> ErrorException: Trying to access array offset on value of type bool
in APPPATH\Controllers\User.php on line 33.
 1 APPPATH\Controllers\User.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type bool', 'C:\\xampp\\htdocs\\serverCw\\app\\Controllers\\User.php', 33)
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:53:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 10:55:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:55:53 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 21.
 1 APPPATH\Models\UserModel.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 21)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:56:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:56:57 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 21.
 1 APPPATH\Models\UserModel.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 21)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:57:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:57:02 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 21.
 1 APPPATH\Models\UserModel.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 21)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:57:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:57:58 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 21.
 1 APPPATH\Models\UserModel.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 21)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:58:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:58:02 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 21.
 1 APPPATH\Models\UserModel.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 21)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:58:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:58:39 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 21.
 1 APPPATH\Models\UserModel.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 21)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 10:59:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 10:59:20 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 21.
 1 APPPATH\Models\UserModel.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 21)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 11:00:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 11:00:09 --> ErrorException: Undefined property: stdClass::$id
in APPPATH\Models\UserModel.php on line 21.
 1 APPPATH\Models\UserModel.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$id', 'C:\\xampp\\htdocs\\serverCw\\app\\Models\\UserModel.php', 21)
 2 APPPATH\Controllers\User.php(30): App\Models\UserModel->verifyUser('kavi', '12')
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\User->login()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 11:00:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 11:05:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 11:06:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 12:24:56 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-13 12:24:57 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-13 12:25:51 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-13 12:33:00 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-13 12:33:00 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-13 12:33:01 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-13 12:33:01 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 12:33:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 12:33:11 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-13 12:36:13 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-13 12:36:13 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-13 12:36:14 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-13 12:36:14 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-13 12:36:22 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-13 12:36:22 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-04-13 12:36:22 --> ErrorException: Undefined variable $questions
in APPPATH\Views\view_home.php on line 99.
 1 APPPATH\Views\view_home.php(99): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $questions', 'C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php', 99)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\serverCw\\app\\Views\\view_home.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1178): CodeIgniter\View\View->render('view_home', [], true)
 5 APPPATH\Controllers\Home.php(13): view('view_home')
 6 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Home->home()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 12:36:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 12:36:29 --> CodeIgniter\HTTP\Exceptions\HTTPException: The route for "/home" cannot be found.
in SYSTEMPATH\HTTP\RedirectResponse.php on line 62.
 1 SYSTEMPATH\HTTP\RedirectResponse.php(62): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('/home')
 2 SYSTEMPATH\Common.php(864): CodeIgniter\HTTP\RedirectResponse->route(false)
 3 APPPATH\Controllers\Questions.php(40): redirect('/home')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Questions->create()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Questions))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 12:36:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 12:37:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 12:37:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 12:38:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 12:38:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 12:39:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 12:52:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 12:52:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 12:52:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 12:52:40 --> CodeIgniter\HTTP\Exceptions\HTTPException: The route for "/home" cannot be found.
in SYSTEMPATH\HTTP\RedirectResponse.php on line 62.
 1 SYSTEMPATH\HTTP\RedirectResponse.php(62): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('/home')
 2 SYSTEMPATH\Common.php(864): CodeIgniter\HTTP\RedirectResponse->route(false)
 3 APPPATH\Controllers\Questions.php(40): redirect('/home')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Questions->create()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Questions))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 12:53:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 12:53:04 --> CodeIgniter\HTTP\Exceptions\HTTPException: The route for "/questions" cannot be found.
in SYSTEMPATH\HTTP\RedirectResponse.php on line 62.
 1 SYSTEMPATH\HTTP\RedirectResponse.php(62): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('/questions')
 2 SYSTEMPATH\Common.php(864): CodeIgniter\HTTP\RedirectResponse->route(false)
 3 APPPATH\Controllers\Questions.php(40): redirect('/questions')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Questions->create()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Questions))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 12:53:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 12:53:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 12:53:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-04-13 12:53:14 --> CodeIgniter\HTTP\Exceptions\HTTPException: The route for "/questions" cannot be found.
in SYSTEMPATH\HTTP\RedirectResponse.php on line 62.
 1 SYSTEMPATH\HTTP\RedirectResponse.php(62): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('/questions')
 2 SYSTEMPATH\Common.php(864): CodeIgniter\HTTP\RedirectResponse->route(false)
 3 APPPATH\Controllers\Questions.php(40): redirect('/questions')
 4 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\Questions->create()
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Questions))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2024-04-13 12:53:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 12:59:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 12:59:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 12:59:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-04-13 12:59:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
